rizinorg/cutter
# Run cutter appimage packaging in GHA.

**Your checklist for this pull request**


**Detailed description**

Move one more configuration from travis to GHA. Configure the Ubuntu 16.04 cutter-deps to do the same as Cutter release package job did in Travis. To simplify things use bundled r2 instead of manually installing r2 system wide in the CI script. Quick test using grep indicates that bundled_r2 build results in some paths to the temporary bundle r2 install directory. Need to investigate if it causes any problems, If there are any problems I would expect most of them to affect the old Appimage the same way.

Rebuild and update Cutter deps. A rebuild was necessary for Linux compatibility with GHA environment. Rebuild also allows compiling using older Visual Studio version fixes the issue with shiboken(clang) not being able to parse latest MSVC headers.    See the corresponding cutter-deps and cutter-deps-qt PRs for more details.

The new MacOS packaging isn't fully enabled as that might have additionall issues and this PR is already a bit too big. I want to leave that for separate PR.

**Test plan (required)**

* Try running resulting artifact on an updated Ubuntu 16.04 VM
* make sure r2 can find all it's resources in share folder
* bundled decompilers work



**Closing issues**
closes #2479  (including latest appveyor failure)
closes #2511
